app/pages/project/stats/stats.jsx

```diff
@@ -273,6 +274,12 @@ export class WorkflowProgress extends React.Component {
         );
       }
       classificationsString += ` / ${total.toLocaleString()}`;
+      // not entirely sure why this first check is needed, but the code crashes without it
+      if (this.props.workflow.configuration) {
+        if (this.props.workflow.configuration.stats_completeness_type === 'classification') {
```

Comment:
The existential operator doesn't exist in ES6. It's been proposed for ES7, but probably not coming any time soon.